### ISSUE TITLE
SVG `<title>` & `<desc>` not allowing interior markup

### ISSUE THREAD

**Issue by Tigt** [validator/validator#104]
[The spec says:](https://developers.whatwg.org/the-map-element.html#svg-0)

>The content model for title elements in the SVG namespace inside HTML documents is phrasing content. (This further constrains the requirements given in the SVG specification.)

I could have sworn I saw something about the namespace autoswitching to HTML in both `<title>` and `<desc>` but I'll be damned if I can find it. [This is the closest I got](https://html.spec.whatwg.org#the-stack-of-open-elements):

>A node is an HTML integration point if it is one of the following elements:
>[...]
>A foreignObject element in the SVG namespace
>A desc element in the SVG namespace
>A title element in the SVG namespace

However, when attempting to use markup (either HTML or SVG tags) inside either `<title>` or `<desc>`, the validator throws an error. This is my test document:

```html
<!DOCTYPE html>
<html>
<head>
<title>Test</title>
</head>
<body>
<svg>
<title>The <a xlink:href="//example.org/">quick brown fox</a> jumped over <em>the lazy dog</em>.</title>
<desc>The <a xlink:href="//example.org/">quick brown fox</a> jumped over <em>the lazy dog</em>.</desc>
</svg>
</body>
</html>
```

**Comment by sideshowbarker** [comment]
Fix pushed to https://validator.w3.org/nu/

As far as your example, the checker is still going to emit an error for the `xlink:href` attribute in there. But that’s appropriate because the HTML `a` element can’t actually have an `xlink:href` attribute.

**Comment by Tigt** [comment]
Oh no, that's desired behavior for sure. I just figured it was a good
negative example, if that makes sense.
